File: code-examples/python/shippings/update.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

payload = "{\"price\" : \"€ 6.00\", \"max_weight\" : 40.00 }"

headers = {
    'authorization': "YourPublicKey:YourToken",
    'accept': "application/json",
    'content-type': "application/json"
    }

conn.request("PUT", "/v0/shippings/:id", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: code-examples/csharp/shippings/update.cs
var client = new RestClient("http://api.marketcloud.it/v0/shippings/:id");
var request = new RestRequest(Method.PUT);
request.AddHeader("content-type", "application/json");
request.AddHeader("accept", "application/json");
request.AddHeader("authorization", "YourPublicKey:YourToken");
request.AddParameter("application/json", "{\"price\" : \"€ 6.00\", \"max_weight\" : 40.00 }", ParameterType.RequestBody);
IRestResponse response = client.Execute(request);